feat(Curriculum) Add A1 Spanish Module 1 (#62201)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/es-a1-practice-greetings-and-farewells/68c98a694e038945907506b2.md

---
id: 68c98a694e038945907506b2
title: Task 5
challengeType: 19
dashedName: task-5
---

<!-- (No Audio) -->

# --instructions--

This task doesn't have audio. Read the question below and select the correct answer.

# --questions--

## --text--

Which of the following options could Camila use to introduce her name?

## --answers--

`Soy` or `Adiós`

### --feedback--

`Adiós` means goodbye. it's not used to introduce yourself.

---

`Me llamo` or `Hasta`

### --feedback--

`Hasta` is part of a farewell, not a way to introduce your name.

---

`Me llamo` or `Soy`

---

`Hasta pronto` or `Me llamo`

### --feedback--

`Me llamo` is correct, but the other phrase is used to say goodbye, not to introduce yourself.

## --video-solution--

3

# --explanation--

> Both `Me llamo` and `Soy` can be used to introduce yourself in Spanish.
> 
> - `Me llamo Camila` – My name is Camila  
> 
> - `Soy Camila` – I am Camila  
> 
> Both are correct, but `Me llamo` is more common when saying your name for the first time.

Reviewer: ```suggestion
Both `Me llamo...` and `Soy...` can be used to introduce yourself in Spanish.

- `Me llamo Camila.` – My name is Camila.

- `Soy Camila.` – I am Camila.

Both are correct, but `Me llamo...` is more common when saying your name for the first time.
```

---
id: 68c98a694e038945907506b2
title: Task 8
challengeType: 19
dashedName: task-8
---

<!-- (No Audio) -->

# --description--

There is more than one way to introduce yourself in Spanish. Can you remember them?

# --instructions--

This task doesn't have audio. Read the question below and select the correct answer.

# --questions--

## --text--

> Which of the following options could you use to introduce your name?

Author: ```suggestion
Which of the following options contain two different ways of introducing your name in Spanish?
```

> Which of the following options contain two different ways of introducing your name in Spanish?

Reviewer 2: ```suggestion
Which of the following options contains two different ways of introducing your name in Spanish?
```
